ui/desktop/package.json

```diff
@@ -42,7 +42,7 @@
     "@ai-sdk/openai": "^0.0.72",
     "@ai-sdk/ui-utils": "^1.0.2",
     "@hey-api/client-fetch": "^0.8.1",
-    "@mcp-ui/client": "^5.3.1",
+    "@mcp-ui/client": "~5.3.1",
```

Comment:
could use https://docs.npmjs.com/cli/v9/configuring-npm/package-json#overrides as well if we want to be very specific

Reply:
im also cool with hardcoding